datasets/international.md

```diff
@@ -0,0 +1,69 @@
+Land cover data from the [Copernicus Land Cover viewer](https://lcviewer.vito.be/2015). Data Commons currently includes observations for US states and counties.
+
+[Citation for Use](https://lcviewer.vito.be/about).
+
+#### [Eurostat Regional Statistics by NUTS Classification](https://ec.europa.eu/eurostat/)
+* [Regions and Cities](https://ec.europa.eu/eurostat/web/regions-and-cities): NUTS (Nomenclature of territorial units for statistics) geocodes, covering NUTS levels 1 through 3.
+* [Demographics (population, age, gender)](https://ec.europa.eu/eurostat/web/population-demography)
+* [Gross domestic product](https://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=nama_10r_3gdp&lang=en): GDP at market prices by NUTS 3 region.
+* Health metrics: Life expectancy, fertility, etc.
+* Education: Education attainment and participation
+
+[Terms of Use](https://ec.europa.eu/eurostat/about/policies/copyright).
+
+#### [India Ministry of Education](https://dashboard.udiseplus.gov.in/#/home)
+Data on schools, such as dropout rate and access to computers and toilets, in Indian states and school districts.
+
+#### [India Ministry of Statistics and Programme Implementation](https://mospi.gov.in/)
+Wage and salary data from the [Periodic Labour Force Survey](https://mospi.gov.in/web/plfs) for Indian states.
+
+#### [National Aeronautics and Space Administration](https://www.nasa.gov/)
+[Center for Climate Simulation](https://www.nccs.nasa.gov/): Earth Exchange Global Daily Downscaled Projections for [CMIP5](https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-gddp) and [CMIP6](https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-gddp-cmip6), which contains atmospheric variables from multiple climate models. Data Commons has aggregated the data for countries and for a 0.5 degree resolution grid, as well as for counties in the US, Europe, and Indian subcontinent.
+
+[Terms of Use](https://www.nccs.nasa.gov/sites/default/files/NEX-GDDP-CMIP6-Tech_Note.pdf).
+
+#### [National Oceanic and Atmospheric Agency](https://www.noaa.gov/)
+[NOAA International Best Track Archive for Climate Stewardship (IBTrACS)](https://www.ncdc.noaa.gov/ibtracs/index.php?name=ib-v4-access): The IBTrACS project provides tropical cyclone best track data in a centralized location. Data Commons includes cyclone name, start date, end date, max wind speed, minimum pressure, max classification, oceanic basin, and affected places.
+
+[Terms of Use](https://www.weather.gov/disclaimer).
+
+#### [Reserve Bank of India](https://rbi.org.in/)
+Data Commons includes variables related to poverty and unemployment in Indian states from the Reserve Bank of India.
+
+#### [UNdata](http://data.un.org/)
+[Population data](http://data.un.org/): for countries, capital cities, urban and rural areas not covered by other sources.
+
+[Terms of Use](http://data.un.org/Host.aspx?Content=UNdataUse).
+
+#### [United Nations](https://www.un.org/)
```

Comment:
I believe this can be grouped with the former UNdata. Perhaps 'United Nations' is the better heading here?

Reply:
done